keyvault/spring-cloud-azure-starter-keyvault-secrets/property-source/README.md

```diff
@@ -1,4 +1,15 @@
-# Spring Cloud Azure Starter Key Vault Secrets Sample 
+---
+page_type: sample
```

Comment:
I remember there is a place to use this metadata, do you know how to use this?

Reply:
@backwind1233 I think it will used by https://learn.microsoft.com/en-us/samples/browse @chenrujun could you please confirm it?

Reply:
Yes. Please check the PR description:
1. 
> ![image](https://user-images.githubusercontent.com/13167207/203699735-77effb84-a902-4be5-b2ae-bc3c33fcfaea.png)

2. 
> ![image](https://user-images.githubusercontent.com/13167207/203699783-30a77a3e-9bf6-456e-a6fc-f12c4745aeb6.png)

Reply:
I'd suggest to make the description less vague.

it takes me sometime to understand the purpose of what is *YAML front-matter*, and even I followed the link, it still not very clear unless I scroll back to the top of the page, it says "Onboarding Samples to learn.microsoft.com/samples"


So, instead of providing a `Add YAML front-matter for property source sample`, how about this:

## Purpose

Onboarding the [property source sample](TODO the link to the specific sample) to https://learn.microsoft.com/samples

Following [this onboarding sample procedure doc]( https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main#supported-metadata-fields-for-readmemd), we need to add [YAML front-matter](https://review.learn.microsoft.com/en-us/help/contribute/samples/process/onboarding?branch=main#supported-metadata-fields-for-readmemd) for property source sample